### PR TITLE
Misc util changes to support raft

### PR DESCRIPTION
**Pull request by tylerwilliams**
Misc util changes to support raft

### CODE REVIEW

File: server/gossip/gossip.go [hunk 1]
```diff
@@ -100,9 +100,9 @@ type logWriter struct {
 func (lw *logWriter) Write(d []byte) (int, error) {
 	s := strings.TrimSuffix(string(d), "\n")
 	if strings.Contains(s, "[DEBUG]") {
-		log.Debugf(s)
+		//		log.Debugf(s)
```
Comment (siggisim) [inline server/gossip/gossip.go:103]: intentional? and below

File: server/util/rangemap/rangemap.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ func (r *Range) Contains(key []byte) bool {
 	lessThanRight := bytes.Compare(key, r.Right) == -1
 
 	contained := greaterThanOrEqualToLeft && lessThanRight
-	log.Debugf("Checking if %s contains %q returned %t", r, string(key), contained)
+	//log.Debugf("Checking if %s contains %q returned %t", r, string(key), contained)
```
Comment (siggisim) [inline server/util/rangemap/rangemap.go:36]: here too

**Review by siggisim — APPROVED** [review]

File: server/util/rangemap/rangemap.go (hunk 2 above)
Comment (tylerwilliams) [reply server/util/rangemap/rangemap.go]: Done.

File: server/gossip/gossip.go (hunk 1 above)
Comment (tylerwilliams) [reply server/gossip/gossip.go]: Done.

**Pull request merged**